datumaro/plugins/kitti_format/importer.py

```diff
@@ -0,0 +1,84 @@
+
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from glob import glob
+import logging as log
+import os.path as osp
+
+from datumaro.components.extractor import Importer
+from datumaro.util.log_utils import logging_disabled
+
+from .format import KittiTask, KittiPath
+
+class KittiImporter(Importer):
+    _TASKS = {
+        KittiTask.segmentation: ('kitti_segmentation', KittiPath.INSTANCES_DIR),
+        KittiTask.detection: ('kitti_detection', KittiPath.LABELS_DIR),
+    }
+
+    @classmethod
+    def detect(cls, path):
+        with logging_disabled(log.WARN):
+            return len(cls.find_sources(path)) != 0
+
+    def __call__(self, path, **extra_params):
+        from datumaro.components.project import Project # cyclic import
+        project = Project()
+
+        subsets = self.find_sources(path)
+
+        if len(subsets) == 0:
+            raise Exception("Failed to find 'kitti' dataset at '%s'" % path)
+
+        # TODO: should be removed when proper label merging is implemented
+        conflicting_types = {KittiTask.segmentation, KittiTask.detection}
```

Comment:
Why is it needed here? Can this format contain multiple unrelated datasets?

Reply:
As you can see [segmentation](https://github.com/openvinotoolkit/datumaro/blob/mz/kitti/datumaro/plugins/kitti_format/format.py#L21) and [detection](https://github.com/openvinotoolkit/datumaro/blob/mz/kitti/datumaro/plugins/kitti_format/format.py#L59) labels are different (in original kitti dataset), so I think, we should avoid usage these both tasks at the same time

Reply:
Ok, got you.